.github/workflows/release-binaries.yml

```diff
@@ -195,7 +195,7 @@ jobs:
   release-windows:
     needs: check-version
     name: Release on Windows ${{ matrix.arch }}
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
Why is this needed?

Reply:
See above "Downgraded Windows VM version so that the code signer will still work."

Reply:
ah didn't read the description. OK.